src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -2946,12 +2976,18 @@ static bool createEntity
                                           ENT_LOCATION_COORDS   << geoJson.obj()));
   }
 
+  /* Add date expiration in the case it was found */
+  if (dateExpiration)
```

Comment:
In line with a previous comment, this could be change to

```
if (dateExpiration != NO_EXPIRATION_DATE)
```

which I think would be clearer.

Reply:
Fixed in faea551